pkg/rehearse/rehearse.go

```diff
@@ -399,25 +382,6 @@ func determineChangedRegistrySteps(candidate, baseSHA string, logger *logrus.Ent
 	return changedRegistrySteps, nil
 }
 
-func determineChangedClusterProfiles(candidate, baseSHA, headSHA string, prNumber int, configUpdaterCfg prowplugins.ConfigUpdater, logger *logrus.Entry) (*ConfigMaps, error) {
```

Comment:
I think we can just delete this function? It is called 'cluster profiles' but it looks like it just deals with config maps, which we no longer need